### ISSUE TITLE
Add missing alt texts and/or aria labels for the main page

### ISSUE THREAD

**Issue by hola-soy-milk** [distributeaid/distributeaid.org#137]
In order to catch these, recommend running Lighthouse on the page, and seeing which images are missing alt text.

[CodeSee Recording](https://app.codesee.io/shared/0b0b10f0-7490-11ec-a628-61bee2c3b61c)
